lib/core/test_result.js

```diff
@@ -79,6 +87,20 @@ class SkippedTest extends TestResult {
   }
 }
 
+class ExplicitlySkippedTest extends TestResult {
+  static canHandle(test, _context) {
+    return test.isExplicitlySkipped();
+  }
+
+  static async handle(test) {
+    test.markExplicitlySkipped(new this());
```

Comment:
We need to change this to `test.finishWithSkippedStatus()` because the result was already set, we just need to trigger the callback